portfolio.html

```diff
@@ -13,7 +13,7 @@
 	</nav>
 	<section class="introSection">
 		<div class="introImage">
-			<img src="tetris.jpg" alt="Man Coding">
+			<img src="tetris.gif" alt="Man Coding">
 		</div>
 		<div class="introTextContainer">
 			<p>Hey There</p>
```

Comment:
Could you use camel case when naming the ID of portfolio (i.e. 'portfolio' rather than 'Portfolio') for consistency?